services/src/main/java/org/fao/geonet/api/registries/vocabularies/KeywordsApi.java

```diff
@@ -1195,6 +1177,113 @@ public String uploadThesaurusFromUrl(
             fname, duration);
     }
 
+
+    /**
+     * Create a new thesaurus.
+     *
+     * @param thesaurusInfo
+     * @param request
+     * @return
+     * @throws Exception
+     */
+    @io.swagger.v3.oas.annotations.Operation(
+        summary = "Creates a new thesaurus",
+        description = "Creates a new thesaurus."
+    )
+    @RequestMapping(
```

Comment:
![image](https://user-images.githubusercontent.com/1701393/213640742-9042e103-c3d6-46fb-9743-a620df7402cc.png)

Usually we use PUT to create something and don't have `ǹew` in path. Should we combine with PUT `/registries/vocabularies` ?